Dockerfile

```diff
@@ -87,3 +88,31 @@ RUN sha256sum --binary ${GOLANG_TEMP} && echo "${GOLANG_SHA256} *${GOLANG_TEMP}"
 RUN tar --extract --gzip --file=${GOLANG_TEMP} --directory=${GOROOT} --strip-components=1
 RUN rm ${GOLANG_TEMP}
 RUN ${GOROOT}/bin/go get github.com/campoy/embedmd
+
+# Install Emscripten.
```

Comment:
I am not sure I understand why we need Emscripten and WABT to be installed in the Docker image, could you clarify? Are they used by the Bazel rules?

Reply:
Emscripten is used by Bazel rules (WABT is not - will delete it).
Currently working on downloading and compiling Emscripten in `WORKSPACE`.

Reply:
It's complicated to load Emscripten in `WORKSPACE`, because SDK contains loaders for Emscripten `./emsdk install 1.39.5 && ./emsdk activate 1.39.5`.